docs/sources/alerting/fundamentals/alert-rule-evaluation/state-and-health.md

```diff
@@ -72,6 +72,12 @@ In [Configure no data and error handling](ref:no-data-and-error-handling), you c
 
 {{< figure src="/media/docs/alerting/alert-rule-configure-no-data-and-error.png" alt="A screenshot of the `Configure no data and error handling` option in Grafana Alerting." max-width="500px" >}}
 
+To reduce the number of no data or error state alerts received, try the following.
+
+1. Use the **Keep last state** option. For more information, refer to the section below.
```

Comment:
```suggestion
1. Use the **Keep last state** option. For more information, refer to the section below. This option allows the alert to retain its last known state when there is no data available, rather than switching to a "No Data" state. 
```